### ISSUE TITLE
Missing DeltaCompressionDotNet.MsDelta.dll

### ISSUE THREAD

**Issue by PhilJenson** [Squirrel/Squirrel.Windows#1217]
My previous release is missing DeltaCompressionDotNet.MsDelta.dll which means the next release will not auto update.  Is there anyway I can recover from this without getting all customers to reinstall (which is impossible)?

**Comment by Andrew-Hanlon** [comment]
I haven't tested this, but... if you ensure that that the set of deltas is larger than the full download, then it should ignore the deltas and perform an upgrade from the latest full package.

**Comment by PhilJenson** [comment]
I have tested this and it works. I can't thank you enough. I have given up all hope of recovering from this.

Incidental they were missing because I updated the nuget releases to the latest only to find that Squirrel wont work with the latest releases, so then wound back. It was at this stage the files were not put back in to the Squirrel release.. 

My app now checks these files are present so when I test the release (prior to going live) I will know if they are missing.

Christmas has come early
Thanks Andrew
